site-content/source/modules/ROOT/pages/blog/Configurable-Storage-Ports-and-Why-We-Need-Them.adoc

```diff
@@ -0,0 +1,36 @@
+= Configurable Storage Ports and Why We Need Them
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: December 28, 2021
```

Comment:
Not sure when this can be published but just noting that the date might need to be updated. 👍